src/triggers/default_triggers.py

```diff
@@ -0,0 +1,60 @@
+from typing import Dict
+
+from frozendict import frozendict
+
+from consts import consts, resources
+from triggers.env_trigger import Trigger
+from triggers.olm_operators_trigger import OlmOperatorsTrigger
+
+_default_triggers = frozendict(
+    {
+        "none_platform": Trigger(
+            ("platform", consts.Platforms.NONE), user_managed_networking=True, vip_dhcp_allocation=False
+        ),
+        "vsphere_platform": Trigger(("platform", consts.Platforms.VSPHERE), user_managed_networking=False),
+        "sno": Trigger(
+            ("masters_count", 1),
```

Comment:
can we have it that way?
```suggestion
            condition=("masters_count", 1),
```